sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/ier12-file-detail.ts

```diff
@@ -28,8 +29,7 @@ export class IER12FileDetail implements IER12FileLine {
   studentGivenName?: string;
   studentBirthDate: Date;
   studentGroupCode: "A" | "B";
-  // Analysis pending for the field.
-  studentMaritalStatusCode?: string;
+  studentMaritalStatusCode: "SI" | "SP" | "MA";
```

Comment:
I was wondering if it wouldn't be better to create a Enum for that.

Reply:
I see your point. I was thinking back and forth about creating enum. 

Only thing which stopped me from creating enum was, the enum isn't going to be used anywhere like `ApplicationStatusCode` which was used for a transformation. 

![image](https://github.com/bcgov/SIMS/assets/54600590/2573905c-bd6b-406d-b1d1-6c27ce753876)